VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist

```diff
@@ -85,6 +85,76 @@
 				</dict>
 			</dict>
 		</dict>
+		<key>Elantech TouchPad</key>
+		<dict>
+			<key>IOProbeScore</key>
+			<integer>7000</integer>
+			<key>IOProviderClass</key>
+			<string>ApplePS2MouseDevice</string>
+			<key>IOClass</key>
+			<string>ApplePS2Elan</string>
+			<key>CFBundleIdentifier</key>
+			<string>as.acidanthera.voodoo.driver.PS2Trackpad</string>
+			<key>Platform Profile</key>
+			<dict>
+				<key>Default</key>
+				<dict>
+					<key>ButtonCount</key>
+					<integer>3</integer>
+					<key>Darwin 16+</key>
+					<dict>
+						<key>ApplePreferenceCapability</key>
+						<true/>
+						<key>ApplePreferenceIdentifier</key>
+						<string>com.apple.AppleMultitouchTrackpad</string>
+						<key>MT Built-in</key>
+						<true/>
+						<key>MTHIDDevice</key>
+						<true/>
+						<key>SupportsGestureScrolling</key>
+						<true/>
+						<key>TrackpadEmbedded</key>
+						<true/>
+						<key>TrackpadFourFingerGestures</key>
+						<true/>
+						<key>TrackpadSecondaryClickCorners</key>
+						<true/>
+						<key>TrackpadThreeFingerDrag</key>
+						<false/>
+					</dict>
+					<key>DisableDevice</key>
+					<false/>
+					<key>ForceTouchMode</key>
+					<integer>1</integer>
+					<key>MouseResolution</key>
+					<integer>3</integer>
+					<key>MouseSampleRate</key>
+					<integer>200</integer>
+					<key>ProcessBluetoothMouseStopsTrackpad</key>
+					<true/>
+					<key>ProcessUSBMouseStopsTrackpad</key>
+					<true/>
+					<key>QuietTimeAfterTyping</key>
+					<integer>500000000</integer>
+					<key>ScrollResolution</key>
+					<integer>400</integer>
+					<key>SetHwResolution</key>
+					<true/>
+					<key>TrackpointMultiplierX</key>
```

Comment:
We're missing TrackpointDividerX and TrackpointDividerY here. In addition, the default value for TrackpointMultiplierX/TrackpointMultiplierY is not the same as in VoodooPS2Elan.hpp

@BAndysc Coud you fix it?